### DISCUSSION TITLE
Is this project still being worked on, or has it been abandoned?

### DISCUSSION THREAD

**Discussion by Derasm** [vitest-dev/vscode#158] [General]
As the title says. Last commit was in february, there are 5 open pull requests and dev haven't given others access to the code. Is this still being worked on?

**Comment by zxch3n** [comment]
While the project isn't currently active, it hasn't been abandoned. The code is openly available for you to fork, adapt, and publish your own extension as needed.

Right now, my focus has shifted elsewhere, making it challenging for me to dedicate the necessary time to its development. There are some big issues that need a good chunk of focused time to sort out. I can't do that just yet, but I'll pick it up in a couple of months.

**Reply by ffMathy** [reply]
I just created a ton of PRs for the repo. If you like what you are seeing, you could add me as an official contributor? That could ease the load a bit.

**Comment by MilanKovacic** [comment]
Hi @keroxp,

I've been following your contributions to the project and appreciate your efforts. Given the maintainer's recent update, I'm considering starting a temporary fork to continue development in the meantime.

Would you be interested in collaborating on this? The goal is to tackle some current issues and maybe add new features. We can plan to integrate back with the main repo once the original maintainer resumes.

Thoughts?

**Reply by keroxp** [reply]
Hmm, I believe that someone will pay attention to the project. Forking is OK, however, it's undesirable duplicated package exists in VS Marketplace (I've seen a similar situation). Currently this project is released into store by 'Zixuan Chen' (@zxch3n). If he doesn't have enough time or attention about the project, we have to ask vitest core team to change publisher of VS Marketplace (or at least mark existing one deprecated). There's no additional active maintainer in the project?

**Reply by MilanKovacic** [reply]
@keroxp 

It would definitely be the best if there was one package that was maintained. Currently there are 8 vitest extensions on the marketplace, and none of them are maintained.
For now, I have started https://github.com/MilanKovacic/vitest-visual-studio-code-extension (where i integrated your changes, too), and published it on marketplace: https://marketplace.visualstudio.com/items?itemName=milan-kovacic.vitest-lab

Ideally, we could continue development in the current (official) repository.

**Reply by sailingKieler** [reply]
Hey @MilanKovacic,

I very very much appreciate that you're takin' over responsibility here. ❤️
Would you do us one additional favor and publish your version to [OpenVSX](https://open-vsx.org/), too?

Thank you anyway!

**Reply by MilanKovacic** [reply]
Hi @sailingKieler,
No problem. I will look into it. Currently, I am focusing on getting the extension in a good state as it has not been updated in a while.

**Comment by MilanKovacic** [comment]
Development on the extension has resumed, with a new release coming soon.
